Description: [DFSG] Debian's Mir src:pkg does not ship test shared libs, so disabling those tests
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: not-required, Debian-specific patch

--- a/tests/unit-tests/CMakeLists.txt
+++ b/tests/unit-tests/CMakeLists.txt
@@ -66,7 +66,7 @@
   test_fatal.cpp
   test_fd.cpp
   test_flags.cpp
-  test_shared_library_prober.cpp
+#  test_shared_library_prober.cpp
   test_lockable_callback.cpp
   test_module_deleter.cpp
   test_mir_cookie.cpp
@@ -228,20 +228,20 @@
   mir_discover_tests_with_fd_leak_detection(mir_umock_unit_tests LD_PRELOAD=libumockdev-preload.so.0 G_SLICE=always-malloc G_DEBUG=gc-friendly)
 endif (MIR_RUN_UNIT_TESTS)
 
-add_custom_command(TARGET mir_unit_tests POST_BUILD
-  COMMAND ${CMAKE_COMMAND} -E copy_directory
-  ${CMAKE_CURRENT_SOURCE_DIR}/shared-libraries ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/test-data/shared-libraries
-  COMMENT "Copying test data to build dir..."
-)
-add_library(loadable_test_dso SHARED ${CMAKE_CURRENT_SOURCE_DIR}/shared-libraries/empty_input.c)
-
-set_target_properties(loadable_test_dso PROPERTIES
-  LIBRARY_OUTPUT_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/test-data/shared-libraries
-  COMPILE_FLAGS "-Wno-pedantic"
-  OUTPUT_NAME this-arch
-)
+#add_custom_command(TARGET mir_unit_tests POST_BUILD
+#  COMMAND ${CMAKE_COMMAND} -E copy_directory
+#  ${CMAKE_CURRENT_SOURCE_DIR}/shared-libraries ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/test-data/shared-libraries
+#  COMMENT "Copying test data to build dir..."
+#)
+#add_library(loadable_test_dso SHARED ${CMAKE_CURRENT_SOURCE_DIR}/shared-libraries/empty_input.c)
+
+#set_target_properties(loadable_test_dso PROPERTIES
+#  LIBRARY_OUTPUT_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/test-data/shared-libraries
+#  COMPILE_FLAGS "-Wno-pedantic"
+#  OUTPUT_NAME this-arch
+#)
 
-add_dependencies(mir_unit_tests loadable_test_dso)
+#add_dependencies(mir_unit_tests loadable_test_dso)
 
 mir_add_test(NAME unit-tests-do-not-link-with-GL-or-EGL
       WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/bin
